pkg/cmd/roachtest/test_runner.go

```diff
@@ -926,6 +927,24 @@ func (r *testRunner) runWorker(
 	}
 }
 
+// destroyCluster runs cluster destroy in a goroutine and adds 1 to the wait group.
+// if the cluster is local, the cluster destroy is sequential.
+func (r *testRunner) destroyCluster(
+	clusterDestroyWg *sync.WaitGroup, c *clusterImpl, l *logger.Logger,
+) {
+	if c.IsLocal() {
```

Comment:
I'd factor this case out and rename the above func to something like `destroyClusterAsync`. Otherwise, there is a confounding factor. The above comment (L666) says, "do not wait", yet we end up waiting in case of `c.IsLocal()`. Also, the `WaitGroup` is coupled with the case which doesn't need it, since it's a blocking call.

Reply:
makes sense! I updated the comments. Thanks @srosenberg !